### PR TITLE
Correct README to reflect the defork from mozilla/go-cose

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ A golang library for the [COSE specification][cose-spec]
 
 **Current Release**: [go-cose alpha 1][release-alpha-1] 
 
-While the project is *currently* forked from the  upstream [mozilla/go-cose][mozilla-go-cose] project, the veraison and mozilla maintainers have agreed to decouple the fork, retire the mozilla/go-cose project and focus on [veraison/go-cose][veraison-go-cose] as the active project.
+The project was *initially* forked from the  upstream [mozilla/go-cose][mozilla-go-cose] project, however it was agreed between the veraison and mozilla maintainers to retire mozilla/go-cose project and focus on [veraison/go-cose][veraison-go-cose] as the active project.
```
Comment (thomas-fossati) [inline README.md:13]:
```suggestion
The project was *initially* forked from the upstream [mozilla-services/go-cose][mozilla-go-cose] project, however the Veraison and Mozilla maintainers have agreed to retire mozilla-services/go-cose and focus on [veraison/go-cose][veraison-go-cose] as the active project.
```

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ A golang library for the [COSE specification][cose-spec]
...
 
 We thank the [mozilla maintainers and contributors][mozilla-contributors] for their great work that formed the base of the [veraison/go-cose][veraison-go-cose] project.
```
Comment (thomas-fossati) [inline README.md:15]:
```suggestion
We thank the [Mozilla maintainers and contributors][mozilla-contributors] for their great work that formed the base of the [veraison/go-cose][veraison-go-cose] project.
```

**Review by thomas-fossati — COMMENTED** [review]
couple of grammar fixes

File: README.md [hunk 3]
```diff
@@ -10,9 +10,9 @@ A golang library for the [COSE specification][cose-spec]
 
 **Current Release**: [go-cose alpha 1][release-alpha-1] 
 
-The project was *initially* forked from the  upstream [mozilla/go-cose][mozilla-go-cose] project, however it was agreed between the veraison and mozilla maintainers to retire mozilla/go-cose project and focus on [veraison/go-cose][veraison-go-cose] as the active project.
+The project was *initially* forked from the  upstream [mozilla-services/go-cose][mozilla-go-cose] project, however it was agreed between the Veraison and Mozilla maintainers to retire mozilla-services/go-cose project and focus on [veraison/go-cose][veraison-go-cose] as the active project.
```
Comment (thomas-fossati) [inline README.md:13]: I prefer using the active phrasing (i.e., as it was before) to the passive construction.

Comment (yogeshbdeshpande) [reply README.md:13]: Nothing wrong in introducing with a brief background of the initial state (earlier) and then thanking the original contributors (in present) and then moving on. In my view it flows well.

Comment (thomas-fossati) [reply README.md:13]: what I am referring to is the change from the original:
```
the veraison and mozilla maintainers have agreed to [...]
```
to
```
it was agreed between the Veraison and Mozilla maintainers to [...]
```
(the former is better.)

Comment (yogeshbdeshpande) [reply README.md:13]: But that was correct when the action was in progress, like I have agreed to hand over XYZ to you.

But once the handover happened that should also go to past tense, like It was agreed between UK and EU to part ways back in 2019.

Comment (thomas-fossati) [reply README.md:13]: But it's not that the agreement has vanished.  Or so I hope :-)

File: README.md [hunk 4]
```diff
@@ -10,9 +10,9 @@ A golang library for the [COSE specification][cose-spec]
 
 **Current Release**: [go-cose alpha 1][release-alpha-1] 
 
-While the project is *currently* forked from the  upstream [mozilla/go-cose][mozilla-go-cose] project, the veraison and mozilla maintainers have agreed to decouple the fork, retire the mozilla/go-cose project and focus on [veraison/go-cose][veraison-go-cose] as the active project.
+The project was *initially* forked from the  upstream [mozilla-services/go-cose][mozilla-go-cose] project, however the Veraison and Mozilla maintainers have agreed to retire mozilla-services/go-cose project and focus on [veraison/go-cose][veraison-go-cose] as the active project.
```
Comment (thomas-fossati) [inline README.md:13]:
```suggestion
The project was *initially* forked from the upstream [mozilla-services/go-cose][mozilla-go-cose] project, however the Veraison and Mozilla maintainers have agreed to retire mozilla-services/go-cose and focus on [veraison/go-cose][veraison-go-cose] as the active project.
```

Comment (yogeshbdeshpande) [reply README.md:13]: I think the existing version carried with project term is well formed. I will retain it like this!

Comment (thomas-fossati) [reply README.md:13]: ok, cool.  Then add a "the" after "retire"

Comment (yogeshbdeshpande) [reply README.md:13]: Done!

**Comment by yogeshbdeshpande** [comment]
@SteveLasker & @thomas-fossati Could you please approve the PR. Seems to be in pretty good shape. 

We need to make a release now once this is merged.

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by thomas-fossati — APPROVED** [review]

**Comment by yogeshbdeshpande** [comment]
Thanks @SteveLasker & @thomas-fossati for adding your comments and review.

**Pull request merged**